### PR TITLE
Moved s7_elephant and bear65 into 1 folder

### PR DESCRIPTION
**Pull request by MudkipMao**
Jacky, the designer of S7 Elephant and Bear 65 will have his own folder now, in the event that he runs more products in the future (which is likely).

## Description

The 2 boards are now moved into the folder jacky_studio/
I have checked with Jacky, and he is ok with this naming convention

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by zvecr — CHANGES_REQUESTED** [review]
Keyboard moves are currently considered a "breaking change" and will need to target develop. Also build commands within readmes should be updated.

**Comment by MudkipMao** [comment]
Holy moly, I did it right this time!

**Comment by drashna** [comment]
> Holy moly, I did it right this time! I rebased my branch against develop

Glad to hear it!  And yeah, the first few times can be .... "fun".  After a while, you really git it down. :)

**Review by drashna — COMMENTED** [review]
Could you add a GPL2+ compatible license header to all of the `c` and `h` files? Including the keymap.c files? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/jacky_studio/s7_elephant/rev1/config.h [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xA13B // Hexidecimal A13B will be Jacky's vendor ID
+#define PRODUCT_ID      0x0007
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    jacky
+#define PRODUCT         S7 Elephant
+#define DESCRIPTION     A high-end 70% custom keyboard
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/config.h:27]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/keymaps/default/config.h [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#define RGBLIGHT_LIMIT_VAL 80
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/keymaps/default/config.h:17]:
```suggestion
#pragma once

#define RGBLIGHT_LIMIT_VAL 80
```

File: keyboards/jacky_studio/s7_elephant/rev1/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+void led_set_user(uint8_t usb_led) {
+    // Turn LEDs On/Off for Caps Lock
+    if (host_keyboard_leds() & (1 << USB_LED_CAPS_LOCK)) {
+        rgblight_enable_noeeprom();
+        rgblight_sethsv_noeeprom(0, 0, 80);
+    } else {
+        rgblight_sethsv_noeeprom(0, 0, 80);
+        rgblight_disable_noeeprom();
+    }
+}
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/keymaps/default/keymap.c:80]:
```suggestion
bool led_update_user(led_t led_state) {
    // Turn LEDs On/Off for Caps Lock
    if (led_state.caps_lock) {
        rgblight_enable_noeeprom();
        rgblight_sethsv_noeeprom(0, 0, 80);
    } else {
        rgblight_sethsv_noeeprom(0, 0, 80);
        rgblight_disable_noeeprom();
    }
    return false;
}
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 4]
```diff
@@ -0,0 +1,2348 @@
...
+                {"x":5.75, "y":4, "w":6.25},
+                {"label":"Fn", "x":12, "y":4, "w":1.5},
+                {"label":"Ctrl", "x":13.5, "y":4, "w":1.5},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_ansi_small_lshift": {
+            "key_count": 73,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:86]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 5]
```diff
@@ -0,0 +1,2348 @@
...
+                {"x":5.75, "y":4, "w":6.25},
+                {"label":"Fn", "x":12, "y":4, "w":1.5},
+                {"label":"Ctrl", "x":13.5, "y":4, "w":1.5},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_ansi_small_bspc": {
+            "key_count": 73,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:164]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 6]
```diff
@@ -0,0 +1,2348 @@
...
+                {"x":5.75, "y":4, "w":6.25},
+                {"label":"Fn", "x":12, "y":4, "w":1.5},
+                {"label":"Ctrl", "x":13.5, "y":4, "w":1.5},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_ansi_small_lshift_small_bspc": {
+            "key_count": 74,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:242]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 7]
```diff
@@ -0,0 +1,2348 @@
...
+                {"x":5.75, "y":4, "w":6.25},
+                {"label":"Fn", "x":12, "y":4, "w":1.5},
+                {"label":"Ctrl", "x":13.5, "y":4, "w":1.5},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_ansi_1u": {
+            "key_count": 73,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:321]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 8]
```diff
@@ -0,0 +1,2348 @@
...
+                {"label":"Fn", "x":12, "y":4},
+                {"label":"Alt", "x":13, "y":4},
+                {"label":"Ctrl", "x":14, "y":4},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_ansi_1u_small_lshift": {
+            "key_count": 74,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:399]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 9]
```diff
@@ -0,0 +1,2348 @@
...
+                {"label":"Fn", "x":12, "y":4},
+                {"label":"Alt", "x":13, "y":4},
+                {"label":"Ctrl", "x":14, "y":4},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_ansi_1u_small_bspc": {
+            "key_count": 74,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:478]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 10]
```diff
@@ -0,0 +1,2348 @@
...
+                {"label":"Fn", "x":12, "y":4},
+                {"label":"Alt", "x":13, "y":4},
+                {"label":"Ctrl", "x":14, "y":4},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_ansi_1u_small_lshift_small_bspc": {
+            "key_count": 75,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:557]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 11]
```diff
@@ -0,0 +1,2348 @@
...
+                {"label":"Fn", "x":12, "y":4},
+                {"label":"Alt", "x":13, "y":4},
+                {"label":"Ctrl", "x":14, "y":4},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_ansi_7u": {
+            "key_count": 71,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:637]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 12]
```diff
@@ -0,0 +1,2348 @@
...
+                {"x":5, "y":4, "w":7},
+                {"label":"Fn", "x":12, "y":4, "w":1.5},
+                {"label":"Ctrl", "x":13.5, "y":4, "w":1.5},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_ansi_7u_small_lshift": {
+            "key_count": 72,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:713]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 13]
```diff
@@ -0,0 +1,2348 @@
...
+                {"x":5, "y":4, "w":7},
+                {"label":"Fn", "x":12, "y":4, "w":1.5},
+                {"label":"Ctrl", "x":13.5, "y":4, "w":1.5},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_ansi_7u_small_bspc": {
+            "key_count": 72,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:790]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 14]
```diff
@@ -0,0 +1,2348 @@
...
+                {"x":5, "y":4, "w":7},
+                {"label":"Fn", "x":12, "y":4, "w":1.5},
+                {"label":"Ctrl", "x":13.5, "y":4, "w":1.5},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_ansi_7u_small_lshift_small_bspc": {
+            "key_count": 73,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:867]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 15]
```diff
@@ -0,0 +1,2348 @@
...
+                {"x":5, "y":4, "w":7},
+                {"label":"Fn", "x":12, "y":4, "w":1.5},
+                {"label":"Ctrl", "x":13.5, "y":4, "w":1.5},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_ansi_blocker": {
+            "key_count": 72,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:945]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 16]
```diff
@@ -0,0 +1,2348 @@
...
+                {"x":5.75, "y":4, "w":6.25},
+                {"label":"Fn", "x":12, "y":4, "w":1.25},
+                {"label":"Ctrl", "x":13.25, "y":4, "w":1.25},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_ansi_blocker_small_lshift": {
+            "key_count": 73,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:1022]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 17]
```diff
@@ -0,0 +1,2348 @@
...
+                {"x":5.75, "y":4, "w":6.25},
+                {"label":"Fn", "x":12, "y":4, "w":1.25},
+                {"label":"Ctrl", "x":13.25, "y":4, "w":1.25},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_ansi_blocker_small_bspc": {
+            "key_count": 73,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:1100]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 18]
```diff
@@ -0,0 +1,2348 @@
...
+                {"x":5.75, "y":4, "w":6.25},
+                {"label":"Fn", "x":12, "y":4, "w":1.25},
+                {"label":"Ctrl", "x":13.25, "y":4, "w":1.25},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_ansi_blocker_small_lshift_small_bspc": {
+            "key_count": 74,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:1178]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 19]
```diff
@@ -0,0 +1,2348 @@
...
+                {"x":5.75, "y":4, "w":6.25},
+                {"label":"Fn", "x":12, "y":4, "w":1.25},
+                {"label":"Ctrl", "x":13.25, "y":4, "w":1.25},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_ansi_blocker_7u": {
+            "key_count": 71,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:1257]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 20]
```diff
@@ -0,0 +1,2348 @@
...
+                {"label":"Alt", "x":4.5, "y":4, "w":1.5},
+                {"label":"Space", "x":6, "y":4, "w":7},
+                {"label":"Fn", "x":13, "y":4, "w":1.5},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_ansi_blocker_7u_small_lshift": {
+            "key_count": 72,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:1333]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 21]
```diff
@@ -0,0 +1,2348 @@
...
+                {"label":"Alt", "x":4.5, "y":4, "w":1.5},
+                {"label":"Space", "x":6, "y":4, "w":7},
+                {"label":"Fn", "x":13, "y":4, "w":1.5},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_ansi_blocker_7u_small_bspc": {
+            "key_count": 72,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:1410]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 22]
```diff
@@ -0,0 +1,2348 @@
...
+                {"label":"Alt", "x":4.5, "y":4, "w":1.5},
+                {"label":"Space", "x":6, "y":4, "w":7},
+                {"label":"Fn", "x":13, "y":4, "w":1.5},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_ansi_blocker_7u_small_lshift_small_bspc": {
+            "key_count": 73,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:1487]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 23]
```diff
@@ -0,0 +1,2348 @@
...
+                {"label":"Alt", "x":4.5, "y":4, "w":1.5},
+                {"label":"Space", "x":6, "y":4, "w":7},
+                {"label":"Fn", "x":13, "y":4, "w":1.5},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_iso": {
+            "key_count": 73,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:1565]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 24]
```diff
@@ -0,0 +1,2348 @@
...
+                {"x":5.75, "y":4, "w":6.25},
+                {"label":"Fn", "x":12, "y":4, "w":1.5},
+                {"label":"Ctrl", "x":13.5, "y":4, "w":1.5},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_iso_small_bspc": {
+            "key_count": 74,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:1643]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 25]
```diff
@@ -0,0 +1,2348 @@
...
+                {"x":5.75, "y":4, "w":6.25},
+                {"label":"Fn", "x":12, "y":4, "w":1.5},
+                {"label":"Ctrl", "x":13.5, "y":4, "w":1.5},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_iso_1u": {
+            "key_count": 74,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:1722]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 26]
```diff
@@ -0,0 +1,2348 @@
...
+                {"label":"Fn", "x":12, "y":4},
+                {"label":"Alt", "x":13, "y":4},
+                {"label":"Ctrl", "x":14, "y":4},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_iso_1u_small_bspc": {
+            "key_count": 75,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:1801]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 27]
```diff
@@ -0,0 +1,2348 @@
...
+                {"label":"Fn", "x":12, "y":4},
+                {"label":"Alt", "x":13, "y":4},
+                {"label":"Ctrl", "x":14, "y":4},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_iso_7u": {
+            "key_count": 72,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:1881]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 28]
```diff
@@ -0,0 +1,2348 @@
...
+                {"x":5, "y":4, "w":7},
+                {"label":"Fn", "x":12, "y":4, "w":1.5},
+                {"label":"Ctrl", "x":13.5, "y":4, "w":1.5},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_iso_7u_small_bspc": {
+            "key_count": 73,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:1958]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 29]
```diff
@@ -0,0 +1,2348 @@
...
+                {"x":5, "y":4, "w":7},
+                {"label":"Fn", "x":12, "y":4, "w":1.5},
+                {"label":"Ctrl", "x":13.5, "y":4, "w":1.5},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_iso_blocker": {
+            "key_count": 73,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:2036]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 30]
```diff
@@ -0,0 +1,2348 @@
...
+                {"x":5.75, "y":4, "w":6.25},
+                {"label":"Fn", "x":12, "y":4, "w":1.25},
+                {"label":"Ctrl", "x":13.25, "y":4, "w":1.25},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_iso_blocker_small_bspc": {
+            "key_count": 74,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:2114]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 31]
```diff
@@ -0,0 +1,2348 @@
...
+                {"x":5.75, "y":4, "w":6.25},
+                {"label":"Fn", "x":12, "y":4, "w":1.25},
+                {"label":"Ctrl", "x":13.25, "y":4, "w":1.25},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_iso_blocker_7u": {
+            "key_count": 72,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:2193]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/info.json [hunk 32]
```diff
@@ -0,0 +1,2348 @@
...
+                {"label":"Alt", "x":4.5, "y":4, "w":1.5},
+                {"x":6, "y":4, "w":6.75},
+                {"label":"Fn", "x":12.75, "y":4, "w":1.5},
+                {"label":"Left", "x":15, "y":4},
+                {"label":"Down", "x":16, "y":4},
+                {"label":"Right", "x":17, "y":4}
+            ]
+        },
+        "LAYOUT_iso_blocker_7u_small_bspc": {
+            "key_count": 73,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/info.json:2270]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/rules.mk [hunk 33]
```diff
@@ -0,0 +1,25 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/rules.mk:11]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev1/rules.mk [hunk 34]
```diff
@@ -0,0 +1,25 @@
...
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   comment out to disable the options.
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev1/rules.mk:15]:
```suggestion
#   change yes to no to disable
```

File: keyboards/jacky_studio/s7_elephant/rev2/config.h [hunk 35]
```diff
@@ -0,0 +1,79 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xA13B // Hexidecimal A13B will be Jacky's vendor ID
+#define PRODUCT_ID      0x0008 // 8 for S7++
+#define DEVICE_VER      0x0002
+#define MANUFACTURER    Jacky
+#define PRODUCT         Jacky S7 Elephant Rev 2
+#define DESCRIPTION     A high-end 70% custom keyboard
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev2/config.h:27]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev2/rules.mk [hunk 36]
```diff
@@ -0,0 +1,33 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev2/rules.mk:11]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev2/rules.mk [hunk 37]
```diff
@@ -0,0 +1,33 @@
...
+SLEEP_LED_ENABLE = no        # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no             # USB Nkey Rollover
+BACKLIGHT_ENABLE = yes       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes        # Enable keyboard RGB underglow
+MIDI_ENABLE = no             # MIDI support
+UNICODE_ENABLE = no          # Unicode
+BLUETOOTH_ENABLE = no        # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no            # Audio output on port C6
+FAUXCLICKY_ENABLE = no       # Use buzzer to emulate clicky switches
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev2/rules.mk:32]:
```suggestion
BLUETOOTH_ENABLE = no        # Enable Bluetooth
AUDIO_ENABLE = no            # Audio output
```

File: keyboards/jacky_studio/s7_elephant/rev2/info.json [hunk 38]
```diff
@@ -0,0 +1,167 @@
...
+                {"label":"K48 (B7,B4)", "x":11.25, "y":4},
+                {"label":"K49 (B7,F7)", "x":12.25, "y":4},
+                {"label":"K4A (B7,F6)", "x":13.25, "y":4},
+                {"label":"K4B (B7,F5)", "x":14.25, "y":4},
+                {"label":"K4C (B7,F4)", "x":15.25, "y":4},
+                {"label":"K4D (B7,F1)", "x":16.25, "y":4}
+            ]
+        },
+        "LAYOUT_ansi_1u": {
+            "key_count": 73,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev2/info.json:89]:
```suggestion
```

File: keyboards/jacky_studio/s7_elephant/rev2/info.json [hunk 39]
```diff
@@ -0,0 +1,167 @@
+{
+    "keyboard_name": "S7 Elephant Rev2",
+    "url": "https://geekhack.org/index.php?topic=103148.0",
+    "maintainer": "qmk",
+    "width": 17.25,
+    "height": 5,
+    "layouts": {
+        "LAYOUT_all": {
+            "key_count": 75,
```
Comment (fauxpark) [inline keyboards/jacky_studio/s7_elephant/rev2/info.json:9]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by MudkipMao** [comment]
What's the current status of this? I had it ready to merge before, but now it's been closed and reopened. If I fix the conflicts, would this be ready to merge again?

**Comment by tzarc** [comment]
We were in the process of merging the `develop` beach, which was accidentally deleted. We're in the process of reopening all closed PRs -- yours was one of about 80 accidentally closed.

**Comment by fauxpark** [comment]
@MudkipMao yes, please rebase this PR, then it should be ready to merge next cycle.

**Comment by MudkipMao** [comment]
I had to do a --force-with-lease to rebase into develop, but it should be ready to go now

**Review by fauxpark — APPROVED** [review]

**Pull request merged**